src/app/ui/ui.js

```diff
@@ -3,12 +3,21 @@
  * @module ui/UI
  */
 /**
+ <<<<<<< 8bit_music
+ * A basic text button.
+ * @param {*} x
+ * @param {number} y
+ * @param {string} text
+ * @param {*} key object associated with button
+ * @param {*} func
+ =======
  * A basic text button. 
  * @param {number} x
  * @param {number} y 
  * @param {string} text 
  * @param {string} key - object associated with button
  * @param {function} func - on click callback function
+ >>>>>>> develop
```

Comment:
This is the end of the previously marked conflict. This one is simple, just remove the first section (from the `8bit_music` line to the `===`), and remove the marking lines